Formula/brew-php-switcher.rb

```diff
@@ -0,0 +1,36 @@
+class BrewPhpSwitcher < Formula
+  desc "Switch Apache / Valet / CLI configs between PHP versions"
+  homepage "https://github.com/philcook/php-switcher"
+  url "https://github.com/philcook/brew-php-switcher/archive/v2.0.tar.gz"
+  sha256 "c2303b1b1a66ee90ed900c3beabacd6aa4e921dbcad5242e399c45d86899bc88"
+  head "https://github.com/philcook/brew-php-switcher.git"
+
+  bottle :unneeded
+
+  depends_on "php" => :test
+  depends_on "php@5.6" => :test
+  depends_on "php@7.0" => :test
+  depends_on "php@7.1" => :test
+
+  def install
+    bin.install "phpswitch.sh"
+    bin.install_symlink "phpswitch.sh" => "brew-php-switcher"
+    bin.install_symlink "phpswitch.sh" => "bps"
+  end
+
+  def caveats; <<~EOS
+    To run brew php switcher do the following:
+      "brew-php-switcher 7.2".
+
+    You can select php version 5.6, 7.0, 7.1, or 7.2.
+    EOS
+  end
+
+  test do
+    # Ensure that PHP version actually switches
+    system "#{bin}/brew-php-switcher", "5.6", "-s"
```

Comment:
This only ensures that the command succeeds. Can you also check for `php -v` after each switch to see if it's actually the version you're expecting?

Reply:
@SMillerDev I tried adding checks for php -v, but it wasn’t working. As it makes changes to the file system (unlinks and links formulas, etc), is Homebrew doing anything strange to prevent issues?

Reply:
I get the following when trying to run it via `brew test`:

```
Switching to php@7.1
Switching your shell
Error: Operation not permitted @ rb_sysopen - /usr/local/var/homebrew/locks/php@5.6.formula.lock
Error: Operation not permitted @ rb_sysopen - /usr/local/var/homebrew/locks/php@7.0.formula.lock
Error: Operation not permitted @ rb_sysopen - /usr/local/var/homebrew/locks/php@7.1.formula.lock
Error: Operation not permitted @ rb_sysopen - /usr/local/var/homebrew/locks/php.formula.lock
Error: Operation not permitted @ rb_sysopen - /usr/local/var/homebrew/locks/php@7.1.formula.lock
All done!
```

Reply:
It works fine when I run it myself. If I can know how to fix this, I'll more than gladly add the version check.